_includes/getting-started.html

```diff
@@ -1,13 +1,42 @@
-<div class="anchor"><a name="getting-started"></a><h2>Run Meshery</h2></div>
+<!DOCTYPE html>
+<html>
+  <head>
+<style>
+  <style>
+.accordion {
```

Comment:
Do we need another method for showing and hiding content? Given the existing "<div class="hidediv"><div class="innerdiv">", is .accordion and .panel necessary?